### PR TITLE
OSDOCS-7518: Add OCVS note to vSphere docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-7518](https://issues.redhat.com/browse/OSDOCS-7518)

Version(s):
4.14

Link to docs preview:
[VMware vSphere infrastructure requirements](https://66076--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations#installation-vsphere-infrastructure_installing-vsphere-installer-provisioned-network-customizations)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-infrastructure.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ You must install the {product-title} cluster on a VMware vSphere version 7.0 Upd
 {product-title} version {product-version} supports VMware vSphere version 8.0.
 ====
 
-You can host the VMware vSphere infrastructure on-premise or on a link:https://cloud.vmware.com/providers[VMware Cloud Verified provider] that meets the requirements outlined in the following table:
+You can host the VMware vSphere infrastructure on-premise or on a link:https://cloud.vmware.com/providers[VMware Cloud Verified provider] that meets the requirements outlined in the following tables:
```
Comment (dfitzmau) [inline modules/installation-vsphere-infrastructure.adoc:21]: Moved note further up, as listing multiple notes does not lead to a good customer UX.

File: modules/installation-vsphere-infrastructure.adoc [hunk 2]
```diff
@@ -29,6 +29,11 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 |vCenter host   | 7.0 Update 2 or later
 |===
 
+[IMPORTANT]
+====
+You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
```
Comment (dfitzmau) [inline modules/installation-vsphere-infrastructure.adoc:34]: Moved note further up, as listing multiple notes does not lead to a good customer UX.

File: modules/installation-vsphere-infrastructure.adoc [hunk 3]
```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 
 [IMPORTANT]
 ====
-You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
+To ensure the best performance conditions for your cluster workloads that operate on OCI and on the Oracle&#174; Cloud VMware Solution (OCVS) service, ensure volume performance units (VPUs) for your block volume is sized for your workloads. 
+
+The following list provides some guidance in selecting the VPUs needed for specific performance needs:
+
+* Test or proof of concept environment: `100` GB, and `20` to `30` VPUs.
+* Basic environment: `500` GB, and `60` VPUs.
```
Comment (mtulio) [inline modules/installation-vsphere-infrastructure.adoc:61]: @racedo "Basic environment" or "Base production environment"?

File: modules/installation-vsphere-infrastructure.adoc [hunk 4]
```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
...
-You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
+To ensure the best performance conditions for your cluster workloads that operate on {oci-first} and on the {ocvs-first} service, ensure volume performance units (VPUs) for your block volume is sized for your workloads.
+
+The following list provides some guidance in selecting the VPUs needed for specific performance needs:
+
+* Test or proof of concept environment: `100` GB, and `20` to `30` VPUs.
+* Basic environment: `500` GB, and `60` VPUs.
+* Heavy-use production environment: More than `500` GB, and `100` or more VPUs.
+
+Consider reserving additional VPUs to give enough capacity for updates and scaling activities.
```
Comment (mtulio) [inline modules/installation-vsphere-infrastructure.adoc:64]: I am not sure if "reserving" is a common word use to in this context, considering VPU is an attribute of the disk which will provide more performance with additional costs.
I can see some alternatives:
- "Consider purchasing": more specific that the customer will pay more for it
- "Consider increasing"
- "Consider allocating"

let me know what do you think @racedo @makentenza

**Review by mtulio — COMMENTED** [review]
nits, overall lgtm. thanks

File: modules/installation-vsphere-infrastructure.adoc (hunk 4 above)
Comment (mtulio) [reply modules/installation-vsphere-infrastructure.adoc:64]: https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm
>  [...] you are purchasing 20 VPUs per GB/month[...] 
> [...] You can purchase more VPUs to allocate more resources to a volume[...]

Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc:64]: Thanks, Marco. "Purchasing" is usually a term exempt from customer docs as it's more towards the marketing side of the business. I can update to "increasing" if that works for everyone?

File: modules/installation-vsphere-infrastructure.adoc (hunk 3 above)
Comment (dfitzmau) [reply modules/installation-vsphere-infrastructure.adoc:61]: Changed to "Base-production environment for now, unless any objections exist?

File: modules/installation-vsphere-infrastructure.adoc [hunk 5]
```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 
 [IMPORTANT]
 ====
-You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
+To ensure the best performance conditions for your cluster workloads that operate on {oci-first} and on the {ocvs-first} service, ensure volume performance units (VPUs) for your block volume is sized for your workloads.
```
Comment (jldohmann) [inline modules/installation-vsphere-infrastructure.adoc:56]:
```suggestion
To ensure the best performance conditions for your cluster workloads that operate on {oci-first} and on the {ocvs-first} service, ensure volume performance units (VPUs) for your block volume are sized for your workloads.
```

File: modules/installation-vsphere-infrastructure.adoc [hunk 6]
```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
...
+
+The following list provides some guidance in selecting the VPUs needed for specific performance needs:
+
+* Test or proof of concept environment: `100` GB, and `20` to `30` VPUs.
```
Comment (jldohmann) [inline modules/installation-vsphere-infrastructure.adoc:60]:
```suggestion
* Test or proof of concept environment: 100 GB, and 20 to 30 VPUs.
```

File: modules/installation-vsphere-infrastructure.adoc [hunk 7]
```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
...
+* Base-production environment: `500` GB, and `60` VPUs.
```
Comment (jldohmann) [inline modules/installation-vsphere-infrastructure.adoc:61]:
```suggestion
* Base-production environment: 500 GB, and 60 VPUs.
```

File: modules/installation-vsphere-infrastructure.adoc [hunk 8]
```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
...
+* Heavy-use production environment: More than `500` GB, and `100` or more VPUs.
```
Comment (jldohmann) [inline modules/installation-vsphere-infrastructure.adoc:62]:
```suggestion
* Heavy-use production environment: More than 500 GB, and 100 or more VPUs.
```

**Review by jldohmann — COMMENTED** [review]
Generally LGTM! some minor nits

**Review by sheriff-rh — APPROVED** [review]
Looks great! Merging.

**Pull request merged**